routes/authors.js

```diff
@@ -1,8 +1,82 @@
 const express = require("express");
 const generateId = require("uuid/v4");
+const authors = require('../data/authors.json')
+const books = require('../data/books.json')
 
 const router = express.Router();
 
 // Define your endpoints here
 
-module.exports = router;
+router.get('/', (req, res, next) => {
+  res.json({
+    authors
```

Comment:
You don't need to enclose the array in an object. You can directly send the array.